rtgui/multilangmgr.cc

```diff
@@ -219,6 +242,77 @@ Glib::ustring MultiLangMgr::getStr (const std::string& key) const
     return key;
 }
 
+const TranslationMetadata *MultiLangMgr::getMetadata(const Glib::ustring &fname) const
+{
+    static const char comment_symbol = '#';
+    static const char *space_chars = " \t";
+    static const char var_symbol = '@';
+    static const char key_value_separator = '=';
+
+    // Look for the metadata in the cache.
+    const auto &found_metadata = lang_files_metadata.find(fname);
+    if (found_metadata != lang_files_metadata.end()) {
+        return &found_metadata->second;
+    }
+
+    std::ifstream file(fname.c_str());
+    if (!file.is_open()) {
+        if (rtengine::settings->verbose) {
+            std::cerr << "Unable to open language file " << fname << " to get metadata." << std::endl;
+        }
+        return nullptr;
+    }
+
+    if (rtengine::settings->verbose) {
+        std::cout << "Reading metadata from language file " << fname << std::endl;
+    }
+    std::map<std::string, std::string> raw_metadata;
+    const auto read_key_value = [&raw_metadata](const std::string &meta_line) {
```

Comment:
@Lawrence37 The following comment (or similar) should also be in the `languages/REAME` file IMHO.